rules/windows/credential_access_spn_attribute_modified.toml

```diff
@@ -0,0 +1,126 @@
+be found [here](https://docs.microsoft.com/en-us/windows-server/security/group-managed-service-accounts/group-managed-service-accounts-overview).
+
+Attackers can also perform "Targeted Kerberoasting", which consists of adding fake SPNs to user accounts that they have
+write privileges to, making them potentially vulnerable to Kerberoasting.
+
+#### Possible investigation steps:
+
+- Identify the account that performed the action.
+- Check whether this user should be doing this kind of activity.
+- Contact the account owner and confirm whether they are aware of this activity.
+- Investigate if the target account is a member of Privileged groups (Domain Admins, Enterprise Admins, etc).
+- Investigate if tickets have been requested for the target account.
+- Investigate other alerts related to the user in the last 48 hours.
+
+### False Positive Analysis
+
+- The use of user accounts as service accounts is a bad security practice and should not be allowed in the domain. The
+security team should map and monitor any potential B-TP (Benign True Positive), especially if the account is privileged.
+Domain Administrators that define this kind of setting can put the domain at risk as user accounts don't have the same
+security standards (Long, complex, random passwords that change frequently) as computer accounts, exposing them to
+credential cracking attacks (Kerberoasting, brute force, etc.).
+
+### Response and Remediation
+
+- Initiate the incident response process based on the outcome of the triage
+- Reset the password of the involved accounts. Priority should be given to privileged accounts.
+- Quarantine the involved host for forensic investigation, as well as eradication and recovery activities.
+
+## Config
+
+The 'Audit Directory Service Changes' logging policy must be configured for (Success, Failure).
+Steps to implement the logging policy with Advanced Audit Configuration:
+
+```
+Computer Configuration > 
+Policies > 
+Windows Settings > 
+Security Settings > 
+Advanced Audit Policies Configuration > 
+Audit Policies > 
+DS Access > 
+Audit Directory Service Changes (Success,Failure)
+```
+
+The above policy does not cover User objects, so we need to set up an AuditRule using https://github.com/OTRF/Set-AuditRule.
+As this specifies the servicePrincipalName Attribute GUID, it is expected to be low noise.
+
+```
+Set-AuditRule -AdObjectPath 'AD:\\CN=Users,DC=Domain,DC=com' -WellKnownSidType WorldSid -Rights WriteProperty -InheritanceFlags Children -AttributeGUID f3a64788-5306-11d1-a9c5-0000f80367c1 -AuditFlags Success
+```
+"""
+references = [
+    "https://www.thehacker.recipes/ad/movement/access-controls/targeted-kerberoasting",
+    "https://www.qomplx.com/qomplx-knowledge-kerberoasting-attacks-explained/",
+    "https://www.thehacker.recipes/ad/movement/kerberos/kerberoast",
+    "https://serverfault.com/questions/541035/kerberos-adding-a-spn-to-a-domain-user",
```

Comment:
```suggestion
```

We can probably remove this link. It doesn't add anything that hasn't been stated by the other links.